packages/gatsby-source-mongodb/README.md

```diff
@@ -26,19 +26,52 @@ Source plugin for pulling data into [Gatsby](https://github.com/gatsbyjs) from M
 * **auth**: the authentication data to login a Mongodb collection, with sub properties user and password.
       ex. auth: { user: `admin`, password: `12345` } 
 
-### TODO WIP
+### Advanced feature
 
-map: with this option you can map a field to a content type, it is an array
-     ex. map: {description: `text\x-markdown`}
+In gatsby you have transformers, that transforms your data for example markdown into html.
+Let's say we have a markdown field in our mongoDB collection. Would it not be wonderfull to use these plugins to render out our content properly.
+
+Well in the gatsby-config.js file you can add the option **map**. 
+With as a key the fieldname and as a value the mediatype of that field, in our case here `text\x-markdown`.
+
+Gatsby will create child nodes for you that will do the magic.
+
+The graphql query would look something like this.
+
+```graphql
+query ItemQuery($id: String!) {
+    mongodbCloudDocuments(id: { eq: $id }) {
+      id
+      name
+      url
+      children {
+          ... on mongodbCloudDocumentsDescriptionMappingNode {
```

Comment:
This works but you should encourage people to query the transformed field through the field itself e.g.

```graphql
mongodbCloudDocuments(id: { eq: $id }) {
  description {
    childMarkdownRemark {
      html
    }
  }
}
```